### PR TITLE
Update introduction.md

### PR DESCRIPTION
**Pull request by fermenreq**
Changes agreed in the meeting

### CODE REVIEW

File: doc/connection-to-the-internet-of-things/introduction.md [hunk 1]
```diff
@@ -32,26 +32,9 @@ implementations:
...
-    MQTT (depends on the compilation options). Github repository is
-    [here](https://github.com/telefonicaid/fiware-IoTAgent-Cplusplus/).
-    FIWARE Lab public instance details are
-    [here](http://catalogue.fiware.org/enablers/backend-device-management-idas/instances). 
--   **IDAS LWM2M:** Supports OMA LightweightM2M over IETF CoAP (IPv4 and
-    IPv6 supported). Github repository is
-    [here](https://github.com/telefonicaid/lightweightm2m-iotagent).
-    FIWARE Lab public instance details are
-    [here](http://catalogue.fiware.org/enablers/backend-device-management-idas/instances).
+-   **[IotAgents](https://github.com/Fiware?utf8=%E2%9C%93&q=IoTAgent):** Acting like translators
```
Comment (jmcanterafonseca) [inline doc/connection-to-the-internet-of-things/introduction.md]: IoT Agents

Comment (jmcanterafonseca) [reply doc/connection-to-the-internet-of-things/introduction.md]: **IoT Agents:** These components stay at the  southbound of the Orion ContextBroker and they are used by IoT integrators to connect devices in this scenario. IoT Agents support several IoT protocols with a modular architecture  Therefore, integrators need to determine first which protocol they will be using to connect devices and select the right IoT Agent.

File: doc/connection-to-the-internet-of-things/introduction.md [hunk 2]
```diff
@@ -67,4 +50,4 @@ Things, keep on reading:
 
 If you want to start experimenting and doing hands-on work, have a look at:
 
-- [IDAS GEri - IoT Agent](https://github.com/Fiware/iot.IoTAgent-Cplusplus)
\ No newline at end of file
+- [IDAS GEri - IoT Agent](https://github.com/Fiware/iot.IoTAgent-Cplusplus)
```
Comment (jmcanterafonseca) [inline doc/connection-to-the-internet-of-things/introduction.md]: link is wrong

File: doc/connection-to-the-internet-of-things/introduction.md (hunk 1 above)
Comment (fermenreq) [reply doc/connection-to-the-internet-of-things/introduction.md]: Hi, 

I just update the description like you said

Thanks

File: doc/connection-to-the-internet-of-things/introduction.md (hunk 2 above)
Comment (fermenreq) [reply doc/connection-to-the-internet-of-things/introduction.md]: It's works for me

**Pull request closed without merging**